docs/api.md

```diff
@@ -1151,8 +1151,13 @@ for details of this object
 
 <a name="parserConfiguration"></a>.parserConfiguration(obj)
 ------------
-`parserConfiguration()` allows you to configure yargs. See [yargs-parser's configuration](https://github.com/yargs/yargs-parser#configuration) for
-settings specific to `yargs-parser`.
+`parserConfiguration()` allows you to configure yargs-parser. 
```

Comment:
let's make this:

"`parserConfiguration()` allows you to configure advanced yargs features:"

Reply:
done.